.github/workflows/deploy.yml

```diff
@@ -82,9 +82,14 @@ jobs:
         python -m pip install --upgrade pip
         pip install --upgrade tox
 
-    - name: Publish GitHub release notes
-      env:
-        GH_RELEASE_NOTES_TOKEN: ${{ github.token }}
+    - name: Generate release notes
       run: |
         sudo apt-get install pandoc
-        tox -e publish-gh-release-notes
+        tox -e generate-gh-release-notes -- ${{ github.event.inputs.version }} scripts/latest-release-notes.md
+
+    - name: Publish GitHub Release
+      uses: softprops/action-gh-release@v1
+      with:
+        body_path: scripts/latest-release-notes.md
+        files: dist/*
```

Comment:
What does this line do (what is in `dist/`)?

Reply:
The sdist and wheel generated by the `hynek/build-and-inspect-python-package@v1.5.4` action.

Reply:
Any particular reason to have these artifacts in github? My instincts tell me it's better to have one source of truth (PyPI).

Reply:
I think it is standard practice, many other plugins follow the same approach.

Note we are publishing exactly the same files that we upload to PyPI, I particularly like it.

Reply:
I am still somewhat interested to know why someone would want to download the wheel/sdist from github instead of pypi, but OK :)

Reply:
Distro packagers do strange things

Reply:
I find it neat to have the sdist/wheel in the GitHub release page -- given it is the exact same package as PyPI I don't see problems with it.